core/domain/feedback_services_test.py

```diff
@@ -134,7 +134,7 @@ def setUp(self):
         # Closed thread without suggestion.
         thread4 = feedback_models.FeedbackThreadModel(
             id=feedback_models.FeedbackThreadModel.generate_full_thread_id(
-                self.EXP_ID2, self.THREAD_ID4),
+                self.EXP_ID1, self.THREAD_ID4),
```

Comment:
I think the original intention here was to use self.EXP_ID2 -- it's the exploration_id below which is wrong. It may not matter though -- but if we're not using EXP_ID2 at all we should probably remove it and rename EXP_ID3 to EXP_ID2.